src/main/java/core/basesyntax/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) throws IOException {
+        return Files.readAllLines(Paths.get(filePath));
+    }
```

Comment:
The method 'readFromFile' should use try-with-resources to ensure that all resources are closed properly. Also, instead of throwing IOException, it's better to catch it and rethrow a RuntimeException with an informative message to maintain a clean interface.

Reply:
Implement this comment